x/cron/codec.proto

```diff
@@ -0,0 +1,21 @@
+syntax = "proto3";
+
+package cron;
+
+import "codec.proto";
+import "gogoproto/gogo.proto";
+
+// TaskResult is a publicly available information about task execution result.
+// It is only created for those tasks that were executed.
+//
+// Due to a bug in tendermint we must store this information ourselves instead
+// of relying on the usual search via tag.
+// https://github.com/tendermint/tendermint/issues/3665
+message TaskResult {
+  weave.Metadata metadata = 1;
+  // Successful is set to true if the task was successfully executed.
+  bool successful = 2;
+  // Info contains any additinal information that might be useful to lean more
+  // about the task execution.
+  string info = 3;
+}
```

Comment:
q: I don't know how this is used later but should we store executed at time/ block as well or rely on the tags?

Reply:
Done in https://github.com/iov-one/weave/pull/850/commits/6e8299bb2eda66cd27b00bf432f05831de74a347